### ISSUE TITLE
Failed to start because of opened port 1900

### ISSUE THREAD

**Issue by murzz** [stefanwichmann/kelvin#5]
First time starting Kelvin, it failed to start. Log and some more info below.
```
$ ./kelvin 
2017/03/31 18:46:42 Kelvin v0.0.11 starting up... 🚀
2017/03/31 18:46:42 Validating local system time...
2017/03/31 18:46:42 Looking for updates...
2017/03/31 18:46:42 Local system time seems to be valid.
2017/03/31 18:46:42 ⚙ Configuration config.json loaded
2017/03/31 18:46:42 ⌘ No bridge configuration found. Starting local discovery...
2017/03/31 18:46:42 listen udp :1900: bind: address already in use
$ netstat -l|grep 1900
udp        0      0 239.255.255.250:1900    0.0.0.0:*                          
$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 16.10
Release:	16.10
Codename:	yakkety
```

**Comment by stefanwichmann** [comment]
This happens when kelvin tries to locally search for hue bridge via UPnP and the needed port is already in use. Currently there are two possible solutions to this:

1) Identify and end the process blocking the port (Try running `netstat -tulpn|grep 1900`. This will give you a PID (process ID) you can use to run `ps -A|grep $PID` to identify the process or run `kill $PID` to end it.)
2) Manually configure the IP and username of your bridge in the `config.json` file. See [Manual bridge configuration](https://github.com/stefanwichmann/kelvin/wiki/Manual-bridge-configuration) for details.

**Comment by stefanwichmann** [comment]
I could work around this bug if I change the behavior of a library Kelvin uses. I'll leave the issue open for this solution.

**Comment by murzz** [comment]
It is `minidlnad` binded to the port. After stopping it Kelvin runs OK.

**Comment by stefanwichmann** [comment]
Just released v0.0.13 which uses a custom fork of the hue library. I made some changes to make bridge discovery more resilient. This should fix this issue (or at least make it far more unlikely).
